Source/main.cpp

```diff
@@ -36,6 +36,10 @@ int main(int argc, char* argv[])
 
     ConvertLabParamsToBoost();
 
+#if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
+    CheckGriddingForRZSpectral();
+#endif
```

Comment:
Could we remove the macros here? They are already present in the function, so let's help readability here.

Reply:
Ok. I was avoiding an unnecessary function call, but removing the macros is fine.